Pod/Classes/LSPAudioViewController.m

```diff
@@ -151,7 +151,6 @@ - (void)play
 {
     DELEGATE_SAFELY(self.delegate, @selector(audioViewController:willPlayPlayer:), [self.delegate audioViewController:self willPlayPlayer:self.player];)
     
-    [self.audioQueuePlayer removeTimeObserver:_progressObserver];
```

Comment:
Maybe we need to change a logic a little bit:
- Doesn't allow to call `play` if audio currently playing. i.e.: `if (self.isPlaying) return;`
- Remove observer in `pause`, `stop` and `reset` methods
- Call `reset` in initializer methods (i.e.:`playAudioWithURL:`)  
So in this case will be clear that we have to observer on `play`